EIPS/eip-7409.md

```diff
@@ -0,0 +1,376 @@
+---
+eip: 7409
+title: Public Non-Fungible Tokens Emote Repository
+description: React to any Non-Fungible Tokens using Unicode emojis.
+author: Bruno Škvorc (@Swader), Steven Pineda (@steven2308), Stevan Bogosavljevic (@stevyhacker), Jan Turk (@ThunderDeliverer)
+discussions-to: https://ethereum-magicians.org/t/eip-6381-emotable-extension-for-non-fungible-tokens/12710
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-26
+requires: 165
+---
+
+
+## Abstract
+
+:information_source: **[ERC-7409](./eip-7409.md) supersedes [ERC-6381](./eip-6381.md).** :information_source:
+The Public Non-Fungible Tokens Emote Repository standard provides an enhanced interactive utility for [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) by allowing NFTs to be emoted at.
```

Comment:
Same as on the other one, but `information_source` doesn't render to anything. Maybe use an emoji?